tools/fix_timepoint_date_problems.php

```diff
@@ -237,12 +237,12 @@ function addInstrument($sessionID, $testName)
     // check the user $_ENV['USER']
     $user =& User::singleton(getenv('USER'));
     if($user->getUsername() == null) {
-        throw new LorisException("Error: Database user named " . getenv('USER') . " does not exist. Please create and then retry script\n");
+        throw new \LorisException("Error: Database user named " . getenv('USER') . " does not exist. Please create and then retry script\n");

```

Comment:
is the `\` to get to the root namespace required or not?

If not, it shouldn't be here, if it is, all the other lines below require it too.